server/migrations/28_add_tenant_id.up.sql

```diff
@@ -0,0 +1,69 @@
+BEGIN;
```

Comment:
You should also create an index for those columns:

```sql
CREATE INDEX idx_config_tenant_id
ON config(tenant_id);
```

Otherwise this will be very costly to query after a few weeks of saas being released.